### ISSUE TITLE
Portal teleport orientation

### ISSUE THREAD

**Issue by KingofAnomaly** [jusdino/BuildPortals#11]
Portals seem to always orient the player to face north when portals are facing north/south And they orient players to the west when the portals are facing east/west no matter how the player walks into the portal. So if I am walking towards the east into a portal it will teleport me and orient my player to the west which is disorienting. The vanilla Minecraft teleport command has an option for the player to keep their orientation on teleportation so I’m sure this is also possible for a plugin to do. Allowing players to keep their orientation will allow players to control how the player walks through the portal and which end they come out of just by building around it.

**Comment by jusdino** [comment]
Yes, that behavior was an intentional design choice. If the player is not rotated when they go through the portal, they could go into a North/South oriented portal then come out of an East/West oriented portal and run straight into the portal pillar. You can make a player come out of the portal facing South or East by partially or completely blocking the North or West side of the portal with solid blocks.

**Comment by KingofAnomaly** [comment]
Ah that makes sense. I didn’t think about having portals oriented differently. I guess that makes things a little more complicated. If you can it though, it would be nice if you can somehow make it so when players go south through a portal they could be oriented south when they come out without having to block them.

**Comment by jusdino** [comment]
Hm... Okay. Let me think about that and if I come up with a good improvement, I'll include it in the next round of updates.

**Comment by KingofAnomaly** [comment]
Ok thank you!
